### DISCUSSION TITLE
Why is BCMath a requirement for laravel?

### DISCUSSION THREAD

**Discussion by mathiasHillmann** [laravel/framework#43816] [General] [answered]
https://laravel.com/docs/9.x/deployment#server-requirements

BCMath stand out as the odd one out of the bunch. Why is it needed?

**Comment by andrey-helldar** [comment, marked as answer]
The framework uses the [`bcmod`](https://www.php.net/manual/en/book.bc.php) function in the [`validator`](https://github.com/laravel/framework/blob/9.x/src/Illuminate/Validation/Concerns/ValidatesAttributes.php#L1466-L1467).

**Reply by xwiz** [reply]
Weird indeed if that's the only use case.

**Reply by andrey-helldar** [reply]
The only one. I searched all BCMath functions across all Laravel Framework 9 files.
